x-pack/plugins/triggers_actions_ui/public/application/sections/action_connector_form/action_type_form.tsx

```diff
@@ -428,7 +428,8 @@ export const ActionTypeForm = ({
     setActionGroupIdByIndex &&
     !actionItem.frequency?.summary;
 
-  const showActionAlertsFilter = hasFieldsForAAD;
+  const showActionAlertsFilter =
+    hasFieldsForAAD || producerId === AlertConsumers.SIEM || hasAlertsMappings;
```

Comment:
it seems to be redundant to have to add again the `hasAlertsMappings` like we do here
https://github.com/elastic/kibana/blob/main/x-pack/plugins/triggers_actions_ui/public/application/sections/rule_form/rule_form.tsx#L655-L660 but it might be safer to do that again. 
```suggestion
    hasFieldsForAAD || producerId === AlertConsumers.SIEM
```